package.json

```diff
@@ -241,6 +241,7 @@
     "openid-client": "^3.15.2",
     "path-to-regexp": "^6.1.0",
     "proper-lockfile": "^4.1.1",
+    "react-router": "^5.2.0",
```

Comment:
This is moved to normal dependencies since required in runtime (`matchPath` is used in page-registry)

Reply:
`react` apparently also required internally in `react-router` when used in runtime